### DISCUSSION TITLE
Equivalent of `brew unlink`?

### DISCUSSION THREAD

**Discussion by bgshacklett** [ScoopInstaller/Scoop#6193] [Help and Questions]
Does Scoop have anything similar to Homebrew's `brew unlink` command? Being able to install (e.g.: java) packages without adding them to the path variable can be helpful for managing multiple versions of a package.

**Comment by HUMORCE** [comment]
https://github.com/ScoopInstaller/Scoop/wiki/Switching-Ruby,-Python-and-PHP-Versions

**Reply by bgshacklett** [reply]
That's helpful, but I don't think it works with an external version manager like jenv, which can handle versioning on a per project basis.
